app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,73 @@
+<resources>
+    <string name="do_install_cuberite">Installeer Cuberite</string>
+    <string name="do_start_cuberite">Start Server</string>
+    <string name="do_stop_cuberite">Stop Server</string>
+    <string name="do_kill_cuberite">Forceer stop Server (Als gewoon stoppen niet werkt)</string>
+    <string name="do_execute_line">Voer uit!</string>
+    <string name="username">Gebruikersnaam</string>
+    <string name="password">Paswoord</string>
+    <string name="enabled">Ingeschakeld</string>
+    <string name="disabled">Uitgeschakeld</string>
```

Comment:
`enabled` and `disabled` are both lowercase in the English version. There could be a reason for that.